cmd/kubeadm/app/util/version.go

```diff
@@ -123,7 +123,14 @@ func splitVersion(version string) (string, string, error) {
 	switch {
 	case strings.HasPrefix(subs[0][2], "ci"):
 		// Special case. CI images populated only by ci-cross area
-		urlSuffix = "ci-cross"
+		if subs[0][3] == "latest" {
```

Comment:
Does anyone use ci/latest-1.11 or just latest-1.11 ?  

I don't know of any tooling that uses ci/latest other then the test infra.

Reply:
@kad you mentioned on slack `I still suggest to simplify it to `urlSuffix = subs[0][2]` in all cases where bucket starts with “ci” and update in test-infra config.json to use ci-cross/latest where it is applicable explicitly.`. So i'll do revise the PR with your suggestion

Reply:
@timothysc `latest-1.11` is usable for official images, `v1.11.0-alpha.2` at the moment. `ci/latest-1.11` points to tip of the branch builds, and images for this build are not available at the moment until corresponding ci-cross build finishes. Yes, primary user is test-infra, and there it will be good to have explicit pointer which build is used for testing (cross vs. standard build).